packages/dids/src/resolver/universal-resolver.ts

```diff
@@ -153,7 +136,6 @@ export class DidResolver {
   /**
    * Dereferences a DID (Decentralized Identifier) URL to a corresponding DID resource.
    *
```

Comment:
Unless the typedoc tooling will complain, consider removing duplicated documentation that is already provided in the interface, VSC automatically infers from the interface.

Reply:
@thehenrytsai This is for the purposes of generating the API reference documentation rather than VSCode.

And to avoid the docs reporting from reporting these:
<img width="896" alt="Screenshot 2024-03-20 at 4 54 18 PM" src="https://github.com/TBD54566975/web5-js/assets/134693/5583dd0a-06fc-400b-a40e-8f33ded86d82">